general/coding_standards.rst

```diff
@@ -42,6 +42,15 @@ Variables are initialized according to these general guidelines:
       which can be a scalar or a composite type are initialized with
       ``memset()`` in order to minimize the amount of future headache.
 
+Unsigned integer constants are defined using the ``U()`` macro or any of
```

Comment:
- By constants you mean what exactly? Macros and global variables? Only macros?
- What about `UL()` and `ULL()`?

Reply:
In practice it's only in defines as in the example below. Variables are not affected by this. What's important here is to make sure that what looks like an unsigned constant really is that too. Exactly which of these macro is chosen isn't that important. The `U()` macro has the advantage of being rather small and works for up to 32-bit values. If a wider width it needed one may choose `UL()` or `ULL()` as appropriate. I guess I mention `UL()` and `ULL()` here.